### PR TITLE
Cordio merge

### PR DESCRIPTION
**Pull request by kevin-gillespie**

### CODE REVIEW

**Comment by kevin-gillespie** [comment]
Running ```git log --follow filename``` will allow us to see the history from the previous repo.

**Review by EdwinFairchild — APPROVED** [review]

**Comment by kevin-gillespie** [comment]
TODO: Revert formatting changes made to Libraries/Cordio.

**Comment by kevin-gillespie** [comment]
There are still a ton of file changes, but they're mostly additions as a result of the submodule going away.

File: Libraries/Cordio/.github/workflows/BLE_Examples_Tests.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        run: |
+          # Remove local modifications
+          git scorch
+
+          # Update the submodules, this will use ssh
+          git submodule init
+          git submodule sync
+          git submodule update --init --recursive
+
+          git submodule deinit -f Libraries/Cordio
```
Comment (sihyung-maxim) [inline Libraries/Cordio/.github/workflows/BLE_Examples_Tests.yml:58]: I think these lines can be deleted now since submodules are completely going away.

**Review by sihyung-maxim — APPROVED** [review]

**Review by sihyung-maxim — APPROVED** [review]

**Comment by kevin-gillespie** [comment]
@EdwinFairchild Does this look like the same encryption failure we debugged?

**Comment by EdwinFairchild** [comment]
No i dont think so, it paired successfully , it just misses some serial output after that and it gets all out of synch. This occasional failed test is only with DATS/DATC connected test which I address in #237

**Pull request merged**